model-optimizer/extensions/back/compress_quantized_weights.py

```diff
@@ -218,3 +218,56 @@ def replace_pattern(self, graph: Graph, match: Dict[str, Node]):
 
         self.quantize_data(fake_quantize, dst_type, quantized_type, mode)
         self.dequantize_data(fake_quantize, dst_type, quantized_type)
+
+
+class ZeroPointOptimizer(BackReplacementPattern):
```

Comment:
Please add description for this transformation: what is zero point, problem and some relevant details